test/diff-test.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+rm -rf ./output
+
+# Diff from schematics
+python ../eagle-diff \
+    --eagle=/Applications/EAGLE-6.6.0/EAGLE.app/Contents/MacOS/EAGLE \
+    ./boards/led_buttons-1/led_buttons-1.sch \
+    ./boards/led_buttons-2/led_buttons-2.sch \
+    ./output
+
+# Diff from boards
+python ../eagle-diff \
+    --force \
+    --eagle=/Applications/EAGLE-6.6.0/EAGLE.app/Contents/MacOS/EAGLE \
```

Comment:
Might be a good idea not to hardcode magic values that are specific to your installation. But you could use e.g. --eagle=eagle to highlight that there is a --eagle switch, but otherwise leave it at its deafult.

Reply:
--eagle=eagle is an _excellent_ suggestion.  I'm commenting to give kudos, but I'm _definitely_ stealing that idea for displaying the option, without necessarily using it!